tools/wasptool

```diff
@@ -364,6 +382,8 @@ if __name__ == '__main__':
             help='Report on the current memory usage.')
     parser.add_argument('--pull',
             help='Fetch a file from the target')
+    parser.add_argument('--force', action='store_true', dest='force_pull',
```

Comment:
I do think that the order of the arguments (alphabetical) is suboptimal and should instead be grouped by topic.

"Force" should be close to "pull" and "as" should be close to "push".